### ISSUE TITLE
Crash running Dedicated Server using RunUnderOneProcess

### ISSUE THREAD

**Issue by OptimisticRaccoon** [fpwong/BlueprintAssistWiki#87]
BlueprintAssistUtils.cpp:2846
`if (TSharedPtr<SViewport> GameViewport = WorldContext.GameViewport->GetGameViewportWidget())`

GameViewportClient.cpp:424
```
  424       TSharedPtr&lt;class SViewport&gt; UGameViewportClient::GetGameViewportWidget() const
  425       {
  426 ***** 	const FSceneViewport* SceneViewport = GetGameViewport(); // null
```

WorldContext of dedicated server has no Viewport and returns null

Steps to replicate:
Editor Preferences|Level Editor|Play
-> tick "Run Under One Process"
-> set Play Net Mode to "Play as Client" or tick Launch Separate Server

Proposed fix: Check if WorldContext is running as a dedicated server
`!WorldContext.RunAsDedicated`, or check if Viewport is valid

**Comment by fpwong** [comment]
Hi @OptimisticRaccoon, thanks I will try to update this asap.

**Comment by fpwong** [comment]
This has been fixed in 4.0.1
